### PR TITLE
Update create-a-phone-system-call-queue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Providing recommended settings for high volume call queues.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-call-queue.md [hunk 1]
```diff
@@ -225,6 +225,15 @@ If an agent opts out of getting calls, they won't be included in the call routin
 > [!IMPORTANT]
 > Agents who use the Skype for Business client aren't included in the call routing list when presence-based routing is enabled, regardless of their availability status. Agents who aren't in the call routing list won't receive calls. If you have agents who use Skype for Business, don't enable presence-based call routing.
 
+> [!IMPORTANT]
+>For high volume calls queues the recommended setttings are:
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:229]:
```suggestion
> For high volume calls queues the recommended setttings are:
```

File: Teams/create-a-phone-system-call-queue.md [hunk 2]
```diff
@@ -225,6 +225,15 @@ If an agent opts out of getting calls, they won't be included in the call routin
...
+>
+>Conference mode: Auto<br>
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:231]:
```suggestion
> Conference mode: Auto<br>
```

File: Teams/create-a-phone-system-call-queue.md [hunk 3]
```diff
@@ -225,6 +225,15 @@ If an agent opts out of getting calls, they won't be included in the call routin
...
+>Routing method: Attendant routing<br>
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:232]:
```suggestion
> Routing method: Attendant routing<br>
```

File: Teams/create-a-phone-system-call-queue.md [hunk 4]
```diff
@@ -225,6 +225,15 @@ If an agent opts out of getting calls, they won't be included in the call routin
...
+>Presence-based routing: On<br>
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:233]:
```suggestion
> Presence-based routing: On<br>
```

File: Teams/create-a-phone-system-call-queue.md [hunk 5]
```diff
@@ -225,6 +225,15 @@ If an agent opts out of getting calls, they won't be included in the call routin
...
+>Agent alert time: 20 seconds
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-call-queue.md:234]:
```suggestion
> Agent alert time: 20 seconds
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**